src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -239,7 +239,9 @@
                 "COLON", "TYPE_EXTENSION_AND").collect(Collectors.toSet()));
         GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("NoWhitespaceBefore", Stream.of(
                 // google uses GenericWhitespace for this behavior
-                "GENERIC_START", "GENERIC_END").collect(Collectors.toSet()));
+                // whitespace is necessary Between a type annotation and ELLIPSIS
```

Comment:
lowercase b in `Between` and please add the rule number that specifies where google allows it. I assume this is 4.6.2 point 9.

Reply:
This comment should only be for ellipsis. We will have different comments for each tokens.

Please also update reasoning as described in next point.

Reply:
done, I update according to the first comment, please give me more information about `Please also update reasoning as described in next point.`

Reply:
@HuGanghui I guess corrected code is not ok since now comment on line 241
`// google uses GenericWhitespace for this behavior` looks weird
I think it was suggested to make it like
```
 // comment
token1, token2,
// another comment
ellipsis
```

Also, it seems that ELLIPSIS should be lowercase

Reply:
ok ,I get it, done